python/pyspark/mllib/linalg/__init__.pyi

```diff
@@ -68,6 +68,7 @@ class Vector:
     __UDT__: VectorUDT
     def toArray(self) -> ndarray: ...
     def asML(self) -> newlinalg.Vector: ...
+    def __len__(self) -> int: ...
```

Comment:
```__len__``` is added to match with logic in 
https://github.com/apache/spark/blob/e29c4e12fbaba542e8150fb71e1c0c1fc15813bd/python/pyspark/mllib/stat/_statistics.py#L268

Reply:
This doesn't look right. ATM `Vector` has not `__len__` method (only subclassess do), so shouldn't have annotation for it.

I believe we should `type: ignore` this for now, and decide if and how to adjust  parent class definitions separately.

Reply:
For the record SPARK-37431

Reply:
thanks! updated with ```ignore```